### PR TITLE
Orgion/future/card implementaion

### PR DESCRIPTION
**Pull request by purushothamanTR**
Card active result list view completed MVC and API with unit testing

### CODE REVIEW

**Review by sathish199308 — COMMENTED** [review]
Comment

File: BookingSystem.Tests/Controller.Tests/CardControllers.Test/GetCardTest.cs [hunk 1]
```diff
@@ -45,32 +44,32 @@ public GetCardTest()
         public async void GetActiveCustomerAsync_ReturnsOkResult()
         {
             // Arrange
-            var carddata = fixture.Create<Task<CardsVM>>();
+            var carddata = fixture.Create<Task<List<CardsVM>>>();
             var expectedResult = await carddata.ConfigureAwait(false);
-            var viewcardData = _testmapper.Map<ViewCardsData>(expectedResult);
- 
-            var expectedId = expectedResult.CardID;
+            var viewcardData = _testmapper.Map<List<ViewCardsData>>(expectedResult);
```
[the remark this answers is not in the document]
Comment (purushothamanTR) [reply BookingSystem.Tests/Controller.Tests/CardControllers.Test/GetCardTest.cs]: _testMapper is used for cardvm to viewcarddatea

File: BookingSystem.Tests/Controller.Tests/CardControllers.Test/GetCardTestById.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    public class GetCardTestById
+    {
+        private readonly Mock<IAPIClientCardRepository> _mockCardRepository;
+        private readonly Mock<IAPIClientCustomerRepository> _mockClientRepository;
+        private readonly Mock<IMapper> _mapper;
+        private static IMapper _testmapper;
+        private readonly Fixture fixture;
+        public GetCardTestById()
+        {
+            if (_testmapper == null)
```
[the remark this answers is not in the document]
Comment (purushothamanTR) [reply BookingSystem.Tests/Controller.Tests/CardControllers.Test/GetCardTestById.cs:28]: _testMapper is used for changes

File: BookingSystem.BAL/Interface/ICardService.cs [hunk 3]
```diff
@@ -11,6 +11,7 @@ namespace BookingSystem.BAL.Interface
     public interface ICardService
     {
         Task<CardsVM> SaveCardasync(CardsVM card);
-        Task<CardsVM> GetActiveCardAsync(int CardID);
+        Task<CardsVM> GetActiveCardByIdAsync(int CardID);
+        List<CardsVM> GetActiveCardAsync();
```
[the remark this answers is not in the document]
Comment (purushothamanTR) [reply BookingSystem.BAL/Interface/ICardService.cs]: changed

**Review by purushothamanTR — COMMENTED** [review]
completed

**Review by sathish199308 — APPROVED** [review]
Card Implementation

**Pull request merged**